spring-cloud-netflix-core/src/main/resources/META-INF/spring.factories

```diff
@@ -20,4 +19,10 @@ org.springframework.cloud.client.discovery.EnableDiscoveryClient=\
 org.springframework.cloud.netflix.eureka.EurekaDiscoveryClientConfiguration
 
 org.springframework.cloud.client.circuitbreaker.EnableCircuitBreaker=\
-org.springframework.cloud.netflix.hystrix.HystrixCircuitBreakerConfiguration
\ No newline at end of file
+org.springframework.cloud.netflix.hystrix.HystrixCircuitBreakerConfiguration
+
+org.springframework.cloud.netflix.metrics.servo.EnableServo=\
```

Comment:
I'm thinking enable via annotations for two reasons:
1.  This implementation is a bit more opinionated -- adds request metrics to MVC endpoints, etc.
2.  We need to be able to make a binary decision between Spectator and Servo.  I *think* that everyone on Java 8 would want Spectator, but I can't presume to understand every use case.